posix.mak

```diff
@@ -374,7 +374,9 @@ clean :
 	rm -rf $(ROOT_OF_THEM_ALL) $(ZIPFILE) $(DOC_OUTPUT_DIR)
 
 zip :
-	zip $(ZIPFILE) $(MAKEFILE) $(ALL_D_FILES) $(ALL_C_FILES) index.d win32.mak win64.mak osmodel.mak
+	zip $(ZIPFILE) $(MAKEFILE) $(ALL_D_FILES) $(ALL_C_FILES) index.d win32.mak win64.mak osmodel.mak \
+	std/range/package.d std/algorithm/package.d std/container/package.d std/regex/package.d \
+	std/experimental/logger/package.d
```

Comment:
We already have a variable for all packages, please use `$(addsuffix /package.d,$(STD_PACKAGES))` instead.